lib/handler.js

```diff
@@ -70,7 +70,15 @@ module.exports = function(options) {
                 actionDescriptionObj.parent = utils.getField(parsedRecord, options.parentField);
               }
 
-              if (options.versionField) {
+              if (options.versionResolver) {
+                const versionResolver = options.versionResolver || (() => {
```

Comment:
I don't think that an alternative path is reachable since we can get here only when `options.versionResolver` is truthy.

Reply:
On a separate note, how about something like:
```js
if (options.versionResolver || options.versionField) {
  const version = options.versionResolver
    ? versionResolver(doc, parsedRecord.OldImage)
    : utils.getField(parsedRecord, options.versionField);
  
  utils.validate(version, schemas.VERSION.label('resolved version'));
  actionDescriptionObj.version = version;
  actionDescriptionObj.versionType = 'external';
}
```

Looks like less duplication. Thoughts?